LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/SiteTroubleshootingView/SiteTroubleshootingView.swift

```diff
@@ -27,6 +27,9 @@ public struct SiteTroubleshootingView: View {
     @Environment(\.colorScheme) var colorScheme
     @EnvironmentObject var model: Model
 
+    @EnvironmentObject
+    private var tipsModel: VPNTipsModel
```

Comment:
Note to self: the tips model should likely be handled by the main models.  Would be cleaner.